spec/actors/hyrax/actors/file_set_actor_spec.rb

```diff
@@ -3,345 +3,350 @@
+            work_v2.ordered_members << create(:file_set)
+            work_v2.save!
+          end
 
-      it "writes to the most up to date version" do
-        actor.attach_to_work(work_v1)
-        expect(work_v1.members.size).to eq 2
+          it "writes to the most up to date version" do
+            actor.attach_to_work(work_v1)
+            expect(work_v1.members.size).to eq 2
+          end
+        end
       end
-    end
-  end
 
-  describe "#file_actor_class" do
-    subject { actor.file_actor_class }
+      describe "#file_actor_class" do
+        subject { actor.file_actor_class }
 
-    it { is_expected.to eq(Hyrax::Actors::FileActor) }
+        it { is_expected.to eq(Hyrax::Actors::FileActor) }
 
-    context "overridden" do
-      let(:custom_fs_actor_class) do
-        class ::CustomFileActor < Hyrax::Actors::FileActor
-        end
-        Class.new(described_class) do
-          def file_actor_class
-            CustomFileActor
+        context "overridden" do
+          let(:custom_fs_actor_class) do
+            class ::CustomFileActor < Hyrax::Actors::FileActor
+            end
+            Class.new(described_class) do
+              def file_actor_class
+                CustomFileActor
+              end
+            end
           end
+          let(:actor) { custom_fs_actor_class.new(file_set, user) }
+
+          after { Object.send(:remove_const, :CustomFileActor) }
+          it { is_expected.to eq(CustomFileActor) }
         end
       end
-      let(:actor) { custom_fs_actor_class.new(file_set, user) }
 
-      after { Object.send(:remove_const, :CustomFileActor) }
-      it { is_expected.to eq(CustomFileActor) }
-    end
-  end
+      describe '#revert_content', perform_enqueued: [IngestJob] do
+        let(:file_set) { create(:file_set, user: user) }
+        let(:file1)    { "small_file.txt" }
+        let(:version1) { "version1" }
+        let(:restored_content) { file_set.reload.original_file }
 
-  describe '#revert_content', perform_enqueued: [IngestJob] do
-    let(:file_set) { create(:file_set, user: user) }
-    let(:file1)    { "small_file.txt" }
-    let(:version1) { "version1" }
-    let(:restored_content) { file_set.reload.original_file }
-
-    before do
-      actor.create_content(fixture_file_upload(file1))
-      actor.create_content(fixture_file_upload('hyrax_generic_stub.txt'))
-      actor.file_set.reload
-    end
+        before do
+          actor.create_content(fixture_file_upload(file1))
+          actor.create_content(fixture_file_upload('hyrax_generic_stub.txt'))
+          actor.file_set.reload
+        end
 
-    it "restores the first versions's content and metadata" do
-      actor.revert_content(version1)
-      expect(restored_content.original_name).to eq file1
+        it "restores the first versions's content and metadata" do
+          pending 'implementation of valkyrized versioning of files' if use_valkyrie
```

Comment:
This is the only change to this spec.  Versioning is not yet implemented using valkyrie, so this test fails when use_valkyrie=true.